packages/data-context/src/actions/WizardActions.ts

```diff
@@ -193,11 +193,12 @@ export class WizardActions {
   private wizardGetConfigCodeE2E (lang: CodeLanguageEnum): string {
     const codeBlocks: string[] = []
 
-    codeBlocks.push(lang === 'ts' ? 'import { defineConfig } from "cypress"' : `const { defineConfig } = require("cypress")`)
+    codeBlocks.push(lang === 'ts' ? `import { defineConfig } from 'cypress'` : `const { defineConfig } = require('cypress')`)
```

Comment:
I was actually see this error with import, (which let me to track down the error stack trace issue I just put out for review) when writing the tests for component setup with Create React App + Typescript. Any idea why?

<img width="833" alt="Screen Shot 2022-01-19 at 11 43 48 AM" src="https://user-images.githubusercontent.com/14099737/150185336-53f70895-481d-4103-8f8d-740ab5ed5d8e.png">

Reply:
Do we need to be validating the package.json type first?